src/DotVVM.Framework/Routing/DotvvmRouteTable.cs

```diff
@@ -77,7 +69,13 @@ public void AddGroup(string groupName, string urlPrefix, string virtualPathPrefi
             virtualPathPrefix = CombinePath(group?.VirtualPathPrefix, virtualPathPrefix);
 
             var newGroup = new DotvvmRouteTable(configuration);
-            newGroup.group = new RouteTableGroup(groupName, group?.RouteNamePrefix + groupName + "_", urlPrefix, virtualPathPrefix, Add);
+            newGroup.group = new RouteTableGroup(
+                groupName: groupName,
+                routeNamePrefix: group?.RouteNamePrefix + groupName + "_",
+                urlPrefix: urlPrefix,
+                virtualPathPrefix: virtualPathPrefix,
+                addToParentRouteTable: Add,
+                presenterFactory: presenterFactory);
```

Comment:
Nit: you can specify only some of the argument names. So you can have `addToParentRouteTable: Add` as it has some information and then only `presenterFactory` as repeating the name twice won't help anyone.